public/components/content-list/_content-list.scss

```diff
@@ -114,7 +115,7 @@ $textualIconSize: 22px;
                 width: 40px;
             }
 
-            &--wordcount {
+            &--printwordcount {
```

Comment:
Slightly puzzled as to why the `wordcount` was duplicated originally?

Reply:
Mistake, I expect.  I took the opportunity to cleanly rectify it.